package.json

```diff
@@ -46,6 +46,8 @@
   },
   "dependencies": {
     "@nomicfoundation/solidity-analyzer": "^0.1.0",
-    "dotenv": "^16.3.1"
+    "bn.js": "^5.2.1",
```

Comment:
Q: shouldn't this result in an update to the package.lock file also?

Reply:
Added.